apps/pool/lib/hooks/useTokensFromPair.ts

```diff
@@ -31,7 +31,7 @@ export function useTokensFromPair(pair: Pair) {
       new Token({
         address: pair.id.includes(':') ? pair.id.split(':')[1] : pair.id,
         name: 'Zenlink LP Token',
-        decimals: pair.chainId === ParachainId.AMPLITUDE ? 12 : 18,
+        decimals: (pair.chainId === ParachainId.AMPLITUDE || pair.chainId === ParachainId.PENDULUM ) ? 12 : 18,
```

Comment:
Would it make sense to extract this `(pair.chainId === ParachainId.AMPLITUDE || pair.chainId === ParachainId.PENDULUM )` check to an extra function and replace all occurences? Having this condition everywhere might be a little too much.

Reply:
That's a fair point. Let's leave it as is then, the condition is long but it's readable. If the Zenlink team wants something different, they'll let us know anyway.